function-aws-api-proxy/src/main/java/io/micronaut/function/aws/proxy/AbstractLambdaContainerHandler.java

```diff
@@ -119,7 +117,7 @@ protected AbstractLambdaContainerHandler(Class<RequestType> requestClass,
      *
      * @return Return the object mapper.
      */
-    protected abstract ObjectMapper objectMapper();
+    protected abstract JsonMapper objectMapper();
```

Comment:
There is a `import io.micronaut.serde.ObjectMapper;`. Why not return it instead?